test/Microsoft.OpenApi.Readers.Tests/ReferenceService/ConvertToOpenApiReferenceV3Tests.cs

```diff
@@ -12,28 +12,19 @@ namespace Microsoft.OpenApi.Readers.Tests.ReferenceService
 {
     public class ConvertToOpenApiReferenceV3Tests
     {
-        private readonly OpenApiV3ReferenceService _referenceService;
-
-        public ConvertToOpenApiReferenceV3Tests()
-        {
-            var context = new ParsingContext();
-            var diagnostic = new OpenApiDiagnostic();
-
-            var yamlDocument = new YamlDocument("{}");
-            var rootNode = new RootNode(context, diagnostic, yamlDocument);
-            _referenceService = new OpenApiV3ReferenceService(rootNode);
-        }
+       
 
         [Fact]
         public void ParseExternalReference()
         {
             // Arrange
+            var referenceService = new OpenApiV3VersionService();
```

Comment:
>referenceService [](start = 16, length = 16)

why still name it as "referenceService"?

Reply:
Because refactoring tools don't catch that.  Will fix.